src/main/java/pl/niewiemmichal/underhiseye/model/Visit.java

```diff
@@ -0,0 +1,54 @@
+package pl.niewiemmichal.underhiseye.model;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@RequiredArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Visit {
+
+    @Id @GeneratedValue
+    private Integer id;
+
+    @Lob
+    private String description;
+
+    @Lob
+    private String diagnosis;
+
+    @NonNull
+    @Column(nullable = false)
+    private Integer status;
+
+    @NonNull
+    @Column(nullable = false)
+    @Temporal(TemporalType.DATE)
+    private Date registrationDate;
+
+    @NonNull
+    @Column(nullable = false)
+    @Temporal(TemporalType.DATE)
+    private Date date;
```

Comment:
can be null

Reply:
We have discussed this via Facebook - visit date cannot be null. Change in data base schema is needed.